rudder-lang/Makefile

```diff
@@ -14,6 +14,10 @@ install:
 	mkdir -p $(DESTDIR)/opt/rudder/share/python
 	mkdir -p $(DESTDIR)/opt/rudder/share/rudder-lang/lib
 	mkdir -p $(DESTDIR)/opt/rudder/share/rudder-lang/tools
+	mkdir -p $(DESTDIR)/var/log/rudder/rudder-lang/
+	
+	# allow for non sudo users to write rudder-lang logs
+	chmod 755 $(DESTDIR)/var/log/rudder/rudder-lang
```

Comment:
these logs should only be readable by the right/group. They may contain configuration and it is sensitive data.